ui/src/testing/mock.rs

```diff
@@ -183,7 +182,22 @@ fn generate_random_identities(count: usize) -> Vec<Identity> {
         let mut graphics = Graphics::default();
         graphics.set_profile_picture(&image_url);
         graphics.set_profile_banner(&image_url);
-
+        
+        let status = match rng.gen_range(0..3) {
+            0 => IdentityStatus::Online,
+            1 => IdentityStatus::Away,
+            2 => IdentityStatus::Busy,
+            _ => IdentityStatus::Offline
+        };
+
+        identity.set_identity_status(status);
+        
+        let platform = match rng.gen() {
```

Comment:
This was originally a numeric generation but since it was always falling out of the scope and hitting `Platform::Unknown` I decided to use a random boolean here for only 2 of the variants in `Platform`